community/emacs/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: stef <l0ls0fo2i@ctrlc.hu>
 # Contributor: Timo Teräs <timo.teras@iki.fi>
 pkgname=emacs
-pkgver=26.2
+pkgver=26.3
 pkgrel=2
```

Comment:
Argh, noticed it too late, but this should've been reset to 0.

Reply:
hrmpf! sorry! i'll try to not forget in the future.

Reply:
No worries, I should've caught that.